src/components/Exercise/List.js

```diff
@@ -1,17 +1,27 @@
 import React from 'react'
 import Exercise from './Exercise'
-import Button from '../common/Button'
+import SearchBox from '../common/SearchBox'
+import { Button } from '../common/Button'
 import { useRouteMatch, Link } from 'react-router-dom'
+import { useForm } from 'react-hook-form'
 import * as S from './List.styles'
 import PropTypes from 'prop-types'
 
 List.propTypes = {
   exercises: PropTypes.array.isRequired,
   handleExerciseSelect: PropTypes.func.isRequired,
+  handleSearch: PropTypes.func.isRequired,
+  search: PropTypes.string,
 }
 
-export default function List({ exercises, handleExerciseSelect }) {
+export default function List({
+  exercises,
+  handleExerciseSelect,
+  handleSearch,
+  search,
+}) {
   const { url } = useRouteMatch()
+  const { register, handleSubmit, reset } = useForm({ mode: 'onChange' })
 
   const lettersAtoZ = [...Array(26)].map((_, i) =>
```

Comment:
Maybe move this outside of the `List` component, it is always constant.

Reply:
nice idea, moved it out of the function body